pages/services/edge-lb/1.2/release-notes/index.md

```diff
@@ -93,6 +93,6 @@ Released on September 11, 2018.
 
 ## Known Issues
 
-* The steps presented in the UI to uninstall Edge-LB are incorrect. Follow the steps in the [Edge-LB uninstall documentation](/services/edge-lb/1.2/uninstalling/).
+* The steps presented in the UI to uninstall Edge-LB are incorrect. Follow the steps in the [Edge-LB uninstall documentation](/services/edge-lb/1.1/uninstalling/).
 * Edge-LB running on a CentOS/RHEL 7.2 node where `/var/lib/mesos` is formatted with ext4 may have connection issues.
-* If a pool is configured with invalid constraints, that pool will not be created or deleted correctly. It must be removed manually.
+* If a pool is configured with invalid constraints, that pool will not be properly created and will not respect pool deletion.  It must be removed manually.
```

Comment:
Maybe:

`If a pool is configured with invalid constraints (e.g. some example if possible) it must be removed manually since a fail on proper creation leads to a failure in proper removal.`